types/chrome/index.d.ts

```diff
@@ -9913,8 +9913,12 @@ declare namespace chrome.tabs {
      * Fired when a tab is zoomed.
      * @since Chrome 38.
      */
-    export var onZoomChange: TabZoomChangeEvent;
-
+    export var onZoomChange: TabZoomChangeEvent;    
```

Comment:
```suggestion
    export var onZoomChange: TabZoomChangeEvent;
```
That's what's causing the CI to fail.